### PR TITLE
fiascotopnm, pnmtofiasco: add pages

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/fiascotopnm.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# fiascotopnm
+
+> Convert a compressed FIASCO file to a PNM image.
+> More information: <https://netpbm.sourceforge.net/doc/fiascotopnm.html>.
+
+- Convert a compressed FIASCO file to one or (in the case of video streams) multiple PNM files:
+
+`fiascotopnm {{path/to/file.fiasco}} -o {{output_file_basename}}`
+
+- Use fast decompression, resulting in a slightly descreased quality of the output file(s):
```
Comment (kbdharun) [inline pages/common/fiascotopnm.md:10]:
```suggestion
- Use fast decompression, resulting in a slightly decreased quality of the output file(s):
```

File: pages/common/fiascotopnm.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# fiascotopnm
+
+> Convert a compressed FIASCO file to a PNM image.
+> More information: <https://netpbm.sourceforge.net/doc/fiascotopnm.html>.
+
+- Convert a compressed FIASCO file to one or (in the case of video streams) multiple PNM files:
```
Comment (kbdharun) [inline pages/common/fiascotopnm.md:6]:
```suggestion
- Convert a compressed FIASCO file to a PNM file or in the case of video streams multiple PNM files:
```
How about this?

File: pages/common/pnmtofiasco.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+> Convert a PNM image to a compressed FIASCO file.
+> More information: <https://netpbm.sourceforge.net/doc/pnmtofiasco.html>.
+
+- Convert a PNM image to a compressed FIASCO file:
+
+`pnmtofiasco {{path/to/file.pnm}} > {{path/to/file.fiasco}}`
+
+- Specify the [i]nput files through a pattern:
+
+`pnmtofiasco -i {{img[01-09+1].pnm}} > {{path/to/file.fiasco}}`
```
Comment (kbdharun) [inline pages/common/pnmtofiasco.md:12]:
```suggestion
`pnmtofiasco --image-name {{img[01-09+1].pnm}} > {{path/to/file.fiasco}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

File: pages/common/pnmtofiasco.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+> Convert a PNM image to a compressed FIASCO file.
+> More information: <https://netpbm.sourceforge.net/doc/pnmtofiasco.html>.
+
+- Convert a PNM image to a compressed FIASCO file:
+
+`pnmtofiasco {{path/to/file.pnm}} > {{path/to/file.fiasco}}`
+
+- Specify the [i]nput files through a pattern:
+
+`pnmtofiasco --image-name {{img[01-09+1].pnm}} > {{path/to/file.fiasco}}`
```
Comment (pixelcmtd) [inline pages/common/pnmtofiasco.md:12]: $ echo pnmtofiasco --image-name img[01-09+1].pnm
zsh: no matches found: img[01-09+1].pnm

```suggestion
`pnmtofiasco --image-name "{{img[01-09+1].pnm}}" > {{path/to/file.fiasco}}`
```

**Pull request merged**